doc/man/scons.xml

```diff
@@ -2109,18 +2109,19 @@ env = Environment()
 <para>&Consenv; attributes called
 <firstterm>&ConsVars;</firstterm> may be set
 either by specifying them as keyword arguments when the object is created
-or by assigning them a value after the object is created:</para>
+or by assigning them a value after the object is created. These two are
+nominally equivalent:</para>
 
 <programlisting language="python">
 env = Environment(FOO='foo')
-env['BAR'] = 'bar'
+env['FOO'] = 'foo'
 </programlisting>
 
-<!--TODO: want to say this, but then have to provide a way for user to tell...
-<para>Certain settings that affect tool detection work
-only on initialization, so check that setting a value
-after creation is appropriate.</para>
--->
+<!--TODO: need to describe a way for a user to tell... -->
+<para>Certain settings that affect tool detection have
+their effect only during initialization, and so need to
+be supplied as part of the calle to &f-Environment;.
```

Comment:
typo calle == call

Reply:
sigh. fixing.  really would like to think of an example, or that paragraph sounds hokey. maybe I'll comment it back out.